source/metadata.py

```diff
@@ -11,10 +11,18 @@
 # This is used by automation like the Pinecil updater
 
 
-if len(sys.argv) != 2:
+if len(sys.argv) < 2:
```

Comment:
If its going to handle more than one argument please indicate the second target in the failure message for future devs.
People are vastly more likely to read the python code than makefile

Reply:
Since it's optional argument for very specific use case, I thought it's not quite necessary to expose any messages at all. But note taken - it makes sense, I think I have idea how to make it right.